Frameworks/Foundation/NSURLSession.mm

```diff
@@ -761,15 +757,21 @@ - (void)downloadTask:(NSURLSessionDownloadTask*)task
                              totalBytesExpectedToWrite);
 }
 
+- (void)_destroyTemporaryFile:(NSURL*)url {
+    [[NSFileManager defaultManager] removeItemAtURL:url error:nullptr];
```

Comment:
wait a second, no. not nil! aj you tricked me!

it's an NSError**, so, nullptr.

---
In reply to: [95448529](https://github.com/Microsoft/WinObjC/pull/1670#discussion_r95448529) [](ancestors = 95448529,95426628)